src/Components/SiteNotFoundPage.tsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from 'react'
+import { connect, load } from 'scrivito'
+import { rootContentId } from '../config/scrivitoSites'
+
+// This page is rendered, if there are no portal sites available.
+export const SiteNotFoundPage = connect(function SiteNotFoundPage() {
+  const [showSiteNotFound, setShowSiteNotFound] = useState(false)
```

Comment:
* [ ] Using state to freeze Scrivito content takes away the ability to rerender in a reactive way. Not sure if and when "Site not found" is shown in editing, but there is a chance that, e.g., fixing an accidental edit will not be reflected in the preview.